JS/Lesson32/savas/my-app/src/App.jsx

```diff
@@ -3,13 +3,15 @@ import { useEffect, useState } from 'react';
 import {BD_SEARCH_BASE_URL} from './urls.js';
 import {RecipeList} from './components/RecipeList';
 import {SearchBar} from './components/SearchBar';
+import { RecipeDetail } from './components/RecipeDetail/index.jsx';
```

Comment:
When a file in a folder is named 'index.js' or 'index.jsx', you don't need to explicitly write it in the import statement. Instead, you can write like this:
```suggestion
import { RecipeDetail } from './components/RecipeDetail';
```